styles/main.css

```diff
@@ -251,7 +251,7 @@ th {
     background-color: rgba(0,0,0,0.1);
 }
 
-tr:nth-of-type(odd) {
+tr:nth-of-type(odd) td {

```

Comment:
If you apply the style to the cells WITHIN the odd-numbered rows, then you don't get the issue of overlapping transparent backgrounds